### ISSUE TITLE
How to change y-axis limit?

### ISSUE THREAD

**Issue by bhishanpdl** [AndreasMadsen/python-lrcurve#5]
I was trying to use this module to plot the training loss and validation loss, but, the default ylim is too wide. How to change the ylim?

# MWE
```
import numpy as np
from lrcurve import PlotLearningCurve

train_loss = [0.48089852, 0.48204786, 0.48174474, 0.4823364 , 0.48186757,
        0.48204969, 0.48138412, 0.48070687, 0.48108125, 0.4794079 ]

 valid_loss = [0.47219259, 0.47824127, 0.47701352, 0.47729106, 0.47710906,
        0.47924276, 0.47544759, 0.48432167, 0.47856586, 0.48392269]

a = min(np.min(train_loss),min(valid_loss))
b = max(np.max(train_loss),max(valid_loss))



with PlotLearningCurve() as plot:
    for i in range(len(train_loss)):
        plot.append(i, {
            'loss': train_loss[i],
            'val_loss': valid_loss[i]
        })
        plot.ylim(a,b)
        plot.draw()
        time.sleep(0.5)

````

**Comment by AndreasMadsen** [comment]
You can use the facet_config argument. https://github.com/AndreasMadsen/python-lrcurve/blob/master/lrcurve/plot_learning_curve.py#L105
